pkg/ebpf/ksyms.go

```diff
@@ -19,23 +20,52 @@ import (
 type existCache struct {
 	mtx  sync.Mutex
 	path string
-	c    map[string]bool
+	c    map[string]uint64
```

Comment:
how many symbols do we store in this map? is the memory overhead from this change neglectable?

Reply:
We only cache the symbols which have been previously recorded. This change only saves the address of the symbol as well now. This would add 8 bytes more for every symbol we cache.
The lock contention telemetry module looks up two symbols, so the cache would hold two extra symbols on top of what we already had. Seems like an acceptable overhead to me.